src/Release/MysqlReleaseGateway.php

```diff
@@ -90,5 +94,23 @@ public function editRelease( $releaseId, $releaseName, $releaseUrl )
         $queryBuilder->execute();
         return true;
     }
+
+    /**
+     * @return Release[]
+     */
+    public function getAllFutureVisibleReleases()
+    {
+        $rows = $this->connection->fetchAll(
+            "SELECT * FROM `release` WHERE `visibility` = 1 AND `release_date` > ?", [(new \DateTime())->format('Y-m-d')]
+        );
+        $releases = [];
+        foreach($rows as $release) {
```

Comment:
Also maybe I'd prefer if this method would take a \DateTime and would be called getReleasesAfter($date) or something. (Don't change it now)